<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>836</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main_icon/main_64.png</normaloff>:/main_icon/main_64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ARMButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
 }
 QPushButton:checked {
     background-color: red;
     border-style: inset;
 }</string>
        </property>
        <property name="locale">
         <locale language="Czech" country="CzechRepublic"/>
        </property>
        <property name="text">
         <string>Záznam</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkNormalize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Normalizuj nahrávku</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTrim">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Odstanit ticho</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkDivide">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Rodělit skladby</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkMP3">
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Převod do MP3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Pracovní adresář</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
          <property name="text">
           <string>Výstupní adresář</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="workDirText">
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="outDirText">
          <property name="locale">
           <locale language="Czech" country="CzechRepublic"/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="workDirBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="outDirBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="MP3Button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Nastavení MP3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Záznamové zařízení</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="devicesCBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="seznam">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Skladba</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Akce</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dokončeno</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
